commons-vfs2/src/main/java/org/apache/commons/vfs2/impl/DefaultFileSystemManager.java

```diff
@@ -686,8 +689,8 @@ public FileObject resolveFile(final FileObject baseFile, final String uri) throw
      * @return A FileObject representing the target file.
      * @throws FileSystemException if an error occurs accessing the file.
      */
-    public FileObject resolveFile(final FileObject baseFile, final String uri,
-            final FileSystemOptions fileSystemOptions) throws FileSystemException {
+    public FileObject resolveFile(  final FileObject baseFile, final String uri, FileSystemOptions fileSystemOptions)
```

Comment:
Formatting issue? Space before the variable?

Reply:
Corrected